NAMESPACE

```diff
@@ -130,9 +130,9 @@ export(array_info)
 export(compute)
 export(configure)
 export(delayed)
-export(delayedArrayUDF)
-export(delayedSQL)
+export(delayed_SQL)
```

Comment:
And what you _could_ consider is to simply offer both:  `delayedSQL` as is and then just assign at the R-level (and add a roxygen2 tag) `delayed_sql <- delayedSQL`.   This flexibility may please some.  It may also upset others who, dunno, feel it intrudes on their TAB-complete (or is that tab_complete now ? :grin:).

Reply:
@eddelbuettel thanks!! No spelling is currently released to users at all so fortunately we don't need to worry about backward compability. :)

Reply:
good call @thetorpedodog :)

Reply:
I am with @thetorpedodog that mixed_Camel_snake is just awful. Please change that.

Reply:
i should have thought of that :)